ui/hooks/useGetFormattedTokensPerChain.ts

```diff
@@ -0,0 +1,82 @@
+import { useSelector } from 'react-redux';
```

Comment:
Can we have a basic test for this hook?

Reply:
Added a test file for it here [04505e4](https://github.com/MetaMask/metamask-extension/pull/28456/commits/04505e46832f3a920986bf1ca165721072a2b8a1)